src/scss/components/dropdown-select/mixins/SelectFacade.scss

```diff
@@ -18,7 +18,7 @@
   }
   @else {
     border: 2px solid $red-900;
```

Comment:
please follow same pattern remaining border properties
change it to border: 2px solid var(--hxDropdown-invalid-border, $red-900);